travis-ymls/kaniko.travis.yml

```diff
@@ -0,0 +1,77 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : Kaniko
+# Source Repo         : https://github.com/GoogleContainerTools/kaniko
+# Travis Job Link     : https://travis-ci.com/github/santosh653/kaniko
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+arch:
+  - amd64
+  - ppc64le
```

Comment:
This changes will not create builds properly as build matrix is defined using `jobs`.